dtmi/azure/iot/diagnosticinformation-1.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@id": "dtmi:azure:iot:diagnosticInformation;1",
+  "@type": "Interface",
+  "description": "Diagnostic Information Interface",
```

Comment:
Should the description follow the guidelines? https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines#interfaces

Reply:
Fixed description

Reply:
Fixed description